energyplus_rmd/test/test_validator.py

```diff
@@ -16,7 +17,7 @@ def test_invalid_dict(self):
         v = Validator()
         passed, message = v.validate_rmd(test2)
         self.assertFalse(passed)
-        self.assertEqual(message, "invalid: 'id' is a required property")
+        self.assertEqual(message, "invalid: 'id' is a required property at $")
```

Comment:
What's the $?

Reply:
It is kind of weird but the validator message includes the dollar sign. It is "err.json_path" maybe it means an empty json_path.